notebook/i18n/README.md

```diff
@@ -15,6 +15,7 @@ However…
 run in).
 
 Hint: if running Windows, you can set it in PowerShell with `${Env:LANG} = "xx_XX"`.
+      if running Ubuntu, you should set environment variable `LANGUAGE="xx_XX"`.
```

Comment:
At least on my system, there's a variable called `LANG` rather than `LANGUAGE`. I think both are used, but I'm not sure which one has priority.

Reply:
on my system(ubuntu14.04)，LANG don't work

Reply:
@forbxy If you are ssh'ing into Ubuntu 14.04, you will need to make sure that the following are set:
```
export LANG="en_US.utf8"
export LANGUAGE="en_US.utf8"
export LC_ALL="en_US.utf8"
```